dietpi/dietpi-survey

```diff
@@ -128,7 +126,7 @@ _EOF_
 -------------------------
 DietPi-Survey v$SURVEY_VERSION
 -------------------------
-Upload Count   : $SURVEY_SENTCOUNT
+Upload Count   : $((SURVEY_SENTCOUNT+1))
```

Comment:
This will be +1 than actual, at end of ```dietpi-update``` which also sends a survey.

Reply:
@Fourdee 
Currently (dietpi-update v6.9) does not call dietpi-survey yet. For this reason, new or pre 6.9 users would upload survey with sentcount=0, as we add +1 after upload, or do I miss something?